test/e2e/providers/kind.go

```diff
@@ -23,6 +24,8 @@ import (
 	"antrea.io/antrea/test/e2e/providers/exec"
 )
 
+var kindKubeconfigPath = flag.String("kind.kubeconfig", path.Join(homedir, ".kube", "config"), "Path of the kubeconfig of the cluster")
```

Comment:
I see that we already have a `remote.kubeconfig` flag for the remote provider, so given that it kind of makes sense. But there is probably some room for improvement in the future.
I was thinking that for our use case it would be simpler to set the KUBECONFIG env variable and make sure the e2e framework can honor its value.

Reply:
Sure we can take up this modification to use kubeconfig env in framework because cuurently it fetches the kubeconfig from providers and in providers the code fore kind returns the config at the homedir, so in case of multiple jobs running at same time they will all start using same cluster so that's why for now I have introduced this flag, later maybe we may consider modifying the framework code.